docs/en/development/CI.md

```diff
@@ -0,0 +1,209 @@
+## CI checks
```

Comment:
* it's better to avoid using the abbreviation in first header and filename
* all headers here and below have one more `#` than they need
* all headers need to be in Title Case